frontend/src/main/components/Jobs/UpdateCoursesByQuarterJobForm.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+
+import { quarterRange } from "main/utils/quarterUtilities";
+
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+
+const UpdateCoursesByQuarterJobForm = ({ callback }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localQuarter = localStorage.getItem("BasicSearch.Quarter");
+
+  const [quarter, setQuarter] = useState(localQuarter || quarters[0].yyyyq);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log("UpdateCoursesJobForm: quarter", quarter);
+    callback({quarter});
+  };
+
+  // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
Same as above... it's ok to invoke this for the part of the code that needs Stryker disabled... but you need to confine this to the smallest possible scope.

Reply:
In https://github.com/ucsb-cs156-f22/f22-5pm-courses/pull/57 I suggest what this might look like.